src/pubkey.h

```diff
@@ -191,6 +175,32 @@ class CPubKey
 
     //! Derive BIP32 child pubkey.
     bool Derive(CPubKey& pubkeyChild, ChainCode &ccChild, unsigned int nChild, const ChainCode& cc) const;
+
+private:
+
+    /**
+     * Just store the serialized data.
+     * Its length can very cheaply be computed from the first byte.
+     */
+    unsigned char vch[65];
```

Comment:
Minor nit, but what about `CPubKey::SIZE` instead of `65` here too?

Reply:
Checked against Bitcoin and then done, thanks